File: codeforces/116A/cpp/116a.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;

    int m = 0, current = 0;
    for (int i = 0; i < n; i++) {
        int leave, enter;
        cin >> leave >> enter;
        current -= leave;
        current += enter;

        m = max(m, current);
    }

    cout << m << endl;

    return 0;
}

File: codeforces/116A/go/116a.go
package main

import "fmt"

func main() {
    var n int
    fmt.Scan(&n)
    var m, cur int
    m = 0
    cur = 0
    for i := 0; i < n; i++ {
        var leave, enter int
        fmt.Scan(&leave, &enter)
        cur -= leave
        cur += enter
        if m < cur {
            m = cur
        }
    }

    fmt.Println(m)
}
